### PR TITLE
Iterations on Branding Page

### PR DESCRIPTION
**Pull request by codiejay**
- [x] Add "Branding" item to footer links. 
- [x] Add new logos and update sections. 

closes https://github.com/EveripediaNetwork/issues/issues/871

### CODE REVIEW

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
new sections looks good
one thing i wanted to point out that i totally forgot about is the quotes section. The texts have a bigger font size in the design. so it shouldn't be the same as the others.
image
Then i also thing that clicking outside the boxes should revert the download options to it's original state
image

spacing between the words are a bit much on mobile and tab
image

**Comment by kesar** [comment]
> Then i also thing that clicking outside the boxes should revert the download options to it's original state

I would also suggest make them look like buttons if they are buttons. We need to train users to understand what buttons are in our site and use them across every single clickable interaction that will produce an effect.

current design looks like a text, not even a link. (a link should be pink w underline effect on hover, and it would be used if you are navigating to a page / url, in this case you are downloading a file)


corners in main image are not rounded like everywhere else. We should keep coherence and avoid squared corners if we use rounded.

image

**Comment by codiejay** [comment]
Fixing these issues now. 
@Simplypheyie96 will you go on to update the design with the new linking UI @kesar is suggesting?

**Comment by Simplypheyie96** [comment]
corners are rounded in the design . for the download button on the page, i felt it would be too colorful having everything in pink since the page itself is so colorful. so i added the text as black with a logo to indicate the download but i'll update that now

**Comment by kesar** [comment]
> corners are rounded in the design . for the download button on the page, i felt it would be too colorful having everything in pink since the page itself is so colorful. so i added the text as black with a logo to indicate the download but i'll update that now
> 
> 

oh i see. we could overlay everything else grey if thats the issue, or as middle point we could try a design w white buttons that i think we have some in other pages. 

up to you, its not a big issue 👍👍

**Comment by Simplypheyie96** [comment]
> > corners are rounded in the design . for the download button on the page, i felt it would be too colorful having everything in pink since the page itself is so colorful. so i added the text as black with a logo to indicate the download but i'll update that now
> > 
> > 
> 
> oh i see. we could overlay everything else grey if thats the issue, or as middle point we could try a design w white buttons that i think we have some in other pages. 
> 
> up to you, its not a big issue 👍👍

an idea that came to me now is to just remove the download text or put them inside the boxes themselves such that a click in the box will show the options to download in svg or png.

**Comment by Simplypheyie96** [comment]
i used pink icons for the white backgrounds and white icons for the rest of the page due to contrast issues. I think it looks neater now compared to before. @kesar @codiejay 
image

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
-change color of download icons to brand color (pink)/ take note of color find dark mode
- The text here on the quotes should have a bigger font size compared to the other texts.
image
then increase spacings between hero and the quotes, then the quotes section and the first sub heading
image
- these texts should also be in pink. check figma
image

**Comment by Simplypheyie96** [comment]
you missed out the last sentence and the 3rd sentence is duplicated @codiejay 
image

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
This should be gray700 in dark mode
image
image
there should be more spacing here
image

**Comment by Simplypheyie96** [comment]
@codiejay additional iterations according to the meeting yesterday is to switch the IQ logo only to the top and bring down the logo with texts. That is, switch sections. 
like this:
image

**Review by Simplypheyie96 — APPROVED** [review]
well done. I think we can merge now

**Comment by Simplypheyie96** [comment]
You can totally merge now @codiejay

File: src/pages/static/branding.tsx [hunk 1]
```diff
@@ -92,8 +81,46 @@ const BrandingPage = () => {
...
+      bg: '/branding/braindaoaltdeepblack.png',
+      download: '/branding/downloadassets/logobraindoadeepblack',
+    },
+  ]
+  const heroImg = useColorModeValue('/brandingBrain.png', '/brandingBrainB.png')
   return (
-    <Box bg="tetiaryGray" _dark={{ backgroundColor: 'tetiaryDark' }} pb={20}>
+    <Box
+      m="0"
+      bg="tetiaryGray"
```
Comment (Adebesin-Cell) [inline src/pages/static/branding.tsx:119]: Use semantic tokens.

File: src/components/Branding/BrandingAssets.tsx [hunk 2]
```diff
@@ -48,24 +64,44 @@ export const BrandingAssets = ({
...
+              setShowDownloadOptions(true)
+            }}
+            fontSize="0px"
+            bg="white"
+            boxShadow="0px 0px 4px 2px rgba(183, 183, 183, 0.21)"
+            variant="outline"
+            border="none"
+            display="flex"
+            borderRadius="200px"
+            m={{ base: '0', lg: '2', md: '2' }}
```
Comment (Adebesin-Cell) [inline src/components/Branding/BrandingAssets.tsx:92]:
```suggestion
            m={{ base: '0', md: '2' }}
```

File: src/components/Branding/BrandingAssets.tsx [hunk 3]
```diff
@@ -48,24 +64,44 @@ export const BrandingAssets = ({
...
+            p="2"
+            w="fit-content"
+            alignItems="center"
+            justifyItems="center"
+            justifyContent="center"
+            rightIcon={
+              <BiCloudDownload fontSize="20px" color={downloadIconColor} />
+            }
+            iconSpacing="0"
+            transform={{ base: 'scale(0.5)', lg: 'scale(1)', md: 'scale(1)' }}
```
Comment (Adebesin-Cell) [inline src/components/Branding/BrandingAssets.tsx:102]:
```suggestion
            transform={{ base: 'scale(0.5)', md: 'scale(1)' }}
```

File: src/components/Branding/BrandingAssetDownloadBttn.tsx [hunk 4]
```diff
@@ -25,7 +25,13 @@ const BrandingAssetDownloadBttn = ({
         width="fit-content"
         onClick={closeDownloadOptions}
       >
-        <Text textAlign="center">{text}</Text>
+        <Text
+          textAlign="center"
+          color="brand.500"
```
Comment (Adebesin-Cell) [inline src/components/Branding/BrandingAssetDownloadBttn.tsx:30]: Use semantic tokens.

**Review by Adebesin-Cell — CHANGES_REQUESTED** [review]

**Review by Adebesin-Cell — APPROVED** [review]
Nice @codiejay.

**Pull request merged**